proto/regen/bond/v1alpha1/types.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+package regen.bond.v1alpha1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/bond";
+
+// BatchInfo represents the high-level on-chain information for a bond.
+message BondInfo {
+
+  // id of the bond
+  string id = 1;
+
+  // emission_denom is the unique id of emission
+  string emission_denom = 2;
+
+  // bond name
+  string name = 3;
+
+  // account address of current holder of the bond
+  string holder = 4;
+
+  // parent bond from which the bond was created by selling to a new holder
+  string parent_bond = 5;
+
+  // face value
+  string face_value = 6;
+
+  // currency of the face value
+  string face_currency = 7;
+
+  // date of bond creation
+  string creation_date = 8;
+
+  // date of issuance
+  string issuance_date = 9;
+
+  // date of maturity
+  string maturity_date = 10;
+
+  // coupon rate
+  string coupon_rate = 11;
+
+  // coupon frequency
+  CouponFrequency coupon_frequency = 12;
+
+  // bond status
+  Status status = 13;
+
+  // project code to which the bond belongs
+  string project = 14;
+
+  // metadata encoded in base64
+  string metadata = 15;
+
+  // Enumeration of project statuses
+  enum Status {
+
+    // Unspecified
+    STATUS_UNSPECIFIED = 0;
+
+    // Active
+    STATUS_ACTIVE = 1;
+
+    // Closed
+    STATUS_CLOSED = 2;
+
+    // Matured
+    STATUS_MATURED = 3;
```

Comment:
I don't think we need to prefix each value name with `STATUS_`: https://developers.google.com/protocol-buffers/docs/proto3#enum

Reply:
I have added the `STATUS_` prefix because the Lint check failed:
```
{"path":"proto/regen/bond/v1alpha1/types.proto","start_line":60,"start_column":5,"end_line":60,"end_column":12,"type":"ENUM_VALUE_PREFIX","message":"Enum value name \"MATURED\" should be prefixed with \"STATUS_\"."} 
```

Reply:
And also in every module in Regen prefixes are used in enum values.